dex/testing/walletpair/walletpair.sh

```diff
@@ -2,19 +2,37 @@
 
 export SHELL=$(which bash)
 SESSION="walletpair"
+
+
+DEXC_ARGS=${@}
+SIMNET="0"
+TESTNET="0"
+while [ "${1:-}" != "" ]; do
+  case "$1" in
+    --simnet)
+      SIMNET="1"
+      echo "Using simnet"
+      ;;
+    --testnet)
+      TESTNET="1"
+      echo "Using testnet"
+      ;;
+  esac
+  shift
+done
+
 if [ "$SIMNET" ] ; then
-  PAIR_ROOT=~/dextest/simnet-walletpair/
+  PAIR_ROOT=~/dextest/simnet-walletpair
```

Comment:
We don't even need a separate directory for simnet. dexc creates network folders anyway. I figured we added the new app dir for simnet because we'll be deleting it a lot.